pkg/integration/tests/patch_building/move_to_new_commit_partial_hunk.go

```diff
@@ -51,14 +51,24 @@ var MoveToNewCommitPartialHunk = NewIntegrationTest(NewIntegrationTestArgs{
 			InitialText(Equals("")).
 			Type("new commit").Confirm()
 
+		t.Views().Commits().
+			IsFocused().
+			Lines(
+				Contains("third commit"),
+				Contains("new commit").IsSelected(),
+				Contains("commit to move from"),
+				Contains("first commit"),
+			).
+			PressEnter()
+
 		t.Views().CommitFiles().
 			IsFocused().
 			Lines(
 				Contains("file1").IsSelected(),
 			).
 			Tap(func() {
 				t.Views().Main().
-					Content(Contains("+1st line\n 2nd line"))
+					Content(Contains("2nd line"))
```

Comment:
Why did you change this? I can drop this change and the test still succeeds. What am I missing?

Reply:
My bad, that line should definitely not change. I'll revert it right now. Thanks for highlighting this, Stefan.